composer.json

```diff
@@ -17,12 +17,16 @@
         }
     ],
     "require": {
-        "symfony/symfony": "~2.3",
-        "sonata-project/admin-bundle": "dev-master",
+        "symfony/framework-bundle": "~2.3",
+        "symfony/twig-bundle": "~2.3",
+        "symfony/security-core": "~2.4",
```

Comment:
@Soullivaneuh the bundle is new. So no need to support SF 2.3 IMO, we can jump to SF2.7